superset/config.py

```diff
@@ -429,6 +429,7 @@ def _try_json_readsha(filepath: str, length: int) -> Optional[str]:
     "GENERIC_CHART_AXES": False,
     "ALLOW_ADHOC_SUBQUERY": False,
     "USE_ANALAGOUS_COLORS": True,
+    "DASHBOARD_EDIT_CHART_IN_TAB": True,
```

Comment:
I can understand the confusion here 😆. I meant `DASHBOARD_EDIT_CHART_IN_NEW_TAB` (added NEW) and this should be `false` by default.

Reply:
And I understood that as `edit in SAME tab` 😆

Reply:
My bad 😆

Reply:
Updated, also updated PR description